### PR TITLE
az-*: update links and page description

### PR DESCRIPTION
**Pull request by sebastiaanspeck**
- [ ] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [ ] The page description(s) have links to documentation or a homepage.
- [ ] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.de/common/az-config.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # az config
 
 > Verwalten der Azure CLI-Konfiguration.
-> Teil von `azure-cli`.
-> Weitere Informationen: <https://learn.microsoft.com/cli/azure/config>.
+> Teil von `az`, der Befehlszeilenschnittstelle von Azure.
```
Comment (gutjuri) [inline pages.de/common/az-config.md:4]:
```suggestion
> Teil von `az`, der Kommandozeilenschnittstelle von Azure.
```

File: pages.de/common/az-logout.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
 # az logout
 
 > Von Azure-Abonnements abmelden.
-> Teil von `azure-cli`.
-> Weitere Informationen: <https://learn.microsoft.com/cli/azure/reference-index#az_logout>.
+> Teil von `az`, der Befehlszeilenschnittstelle von Azure.
```
Comment (gutjuri) [inline pages.de/common/az-logout.md:4]:
```suggestion
> Teil von `az`, der Kommandozeilenschnittstelle von Azure.
```

**Review by gutjuri — CHANGES_REQUESTED** [review]
Thanks you! Looks fine to me, apart from the german word "Befehlszeile", which I changed to "Kommandozeile" in my suggestion.

**Comment by sebastiaanspeck** [comment]
> Thanks you! Looks fine to me, apart from the german word "Befehlszeile", which I changed to "Kommandozeile" in my suggestion.

Changed in all three az-files. Thank you for the suggestion!

**Comment by gutjuri** [comment]
LGTM

**Review by gutjuri — APPROVED** [review]

File: pages.es/common/az-appconfig.md [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
 > Administra configuraciones de aplicaciones en Azure.
```
Comment (kant) [inline pages.es/common/az-appconfig.md:3]: Administra configuraciones --> Administra las configuraciones

File: pages.es/common/az-bicep.md
Comment (kant) [inline pages.es/common/az-bicep.md:1]: Line 11: Crear un archivo --> Crea un archivo

File: pages.es/common/az-feedback.md [hunk 4]
```diff
@@ -1,8 +1,8 @@
 # az feedback
 
 > Envía comentarios al equipo de Azure CLI.
-> Parte de `azure-cli`.
-> Más información: <https://learn.microsoft.com/cli/azure/reference-index#az_feedback>.
+> Parte de `az`, el cliente de línea de comandos para Microsoft Azure.
+> Más información: <https://learn.microsoft.com/es-es/cli/azure/reference-index#az_feedback>.
 
 - Envia comentarios al equipo de Azure CLI:
```
Comment (kant) [inline pages.es/common/az-feedback.md:7]: Envia --> Envía

**Review by acuteenvy — COMMENTED** [review]
There's recently been [a PR](https://github.com/tldr-pages/tldr/pull/10929) removing the "command-line client for Microsoft Azure" part as per the [style guide](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#descriptions).

**Review by kbdharun — CHANGES_REQUESTED** [review]
Thanks for your contribution. As @acuteenvy says, we can drop the `the command-line client for Microsoft Azure` part. But the `azure-cli` -> `az` change LGTM as we have the page named `az` too (I will add an alias page named `azure-cli`).

**Review by kbdharun — CHANGES_REQUESTED** [review]
We don't specify the locale in Microsoft links as it will automatically redirect to the reader's preferred locale (it is especially useful for Bilingual speakers).

**Comment by sebastiaanspeck** [comment]
Want to edit the new az-pages as well.

**Review by gutjuri — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]

**Pull request merged**